### DISCUSSION TITLE
[Help Wanted] Using this forum for issues with derived editors (Rich Text, Markdown, Spreadsheets, Code)

### DISCUSSION THREAD

**Discussion by user4-33** [standardnotes/forum#3200] [General] [answered]
Since the Standardnotes [Website](https://standardnotes.com/help/77/what-are-editors) says that any issues related with external editors (in this case the Summernote Rich Text editor) are not their case - is it even recommended to report bugs for those in the Standardnotes forum?

(The Rich Text editor has some pretty frustrating bugs that decrease the user experience...)

**Comment by effieeee** [comment, marked as answer]
Hmm, you may still report them on the [Discussions](https://github.com/standardnotes/forum/discussions) page, and we'll still be notified about it. There are cases wherein the issue is something we can address on our end, so it wouldn't hurt to bring it up here :)
